connector/spanmetricsconnector/connector.go

```diff
@@ -296,22 +296,48 @@ func (p *connectorImp) buildMetrics() pmetric.Metrics {
 		}
 
 		sums := rawMetrics.sums
+		// Duplicate metric to avoid breaking change:
+		// https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33227
 		metric := sm.Metrics().AppendEmpty()
 		metric.SetName(buildMetricName(p.config.Namespace, metricNameCalls))
 		sums.BuildMetrics(metric, startTimeGenerator, timestamp, p.config.GetAggregationTemporality())
+
+		if p.config.Namespace != "" && p.config.Namespace == DefaultNamespace {
```

Comment:
Can we use a `StageBeta` feature gate `legacyMetricNamesFeatureGate` to determine whether to enable it?
The `StageBeta` feature gate will enable it by default and users can also disable it.

Reply:
Sure. I'll send a PR now.